README.md

```diff
@@ -54,6 +54,10 @@ To build the app from source:
 * Uses ArcGIS Runtime 10.2.7
 * Requires Android 4.0.3 or higher
 * Added Mil2525CMessageLayer layer type for GeoMessage files (see the [User Guide](documentation/UserGuide.md#geomessage-files-and-layers))
+* Refactored the location capabilities
+  * TODO users
```

Comment:
TODOs are fine even in Readme's (I actually wish people would use them more, it is much preferable to potentially forgetting)